R/arguments.R

```diff
@@ -39,7 +39,10 @@ wsc_parse_numeric <- function(opt, varname, val_for_na=NA, length=1){
   }else{
     vals <- wsc_split_string(opt[[varname]])
   }
-  
+  if(grepl("[0-10]+:[0:10]+",opt[[varname]])){
```

Comment:
You need to add this to the conditional above- the wsc_split_string() call should not occur if your conditional matches. So add it as an 'else if {' before that last else.